packages/framework/src/Model/LegalConditions/LegalConditionsFacade.php

```diff
@@ -113,7 +113,7 @@ protected function findArticle($settingKey, $domainId)
         $articleId = $this->setting->getForDomain($settingKey, $domainId);
 
         if ($articleId !== null) {
-            return $this->articleFacade->getById($articleId);
+            return $this->articleFacade->findById($articleId);
```

Comment:
How about to skip if and returning null and return a result of `ArticleFacade::findById` directly?

Reply:
But `ArticleFacade::findById()` does not accept `null` as a parameter, only `int`.

There are three cases:
1. The article is not set.
2. The article is set but does not exists. (this case is fixed here)
3. The article is set and exists.

Reply:
Sorry I forgot that findById does not accept null as an argument. It's okay.